app/models/resource.rb

```diff
@@ -1,10 +1,11 @@
 class Resource < ActiveRecord::Base
   belongs_to :conference
+  validates :name, :quantity, presence: true
   validate :used_less_than_quantity
 
   private
 
   def used_less_than_quantity
-    errors.add(:used, 'can not be higher than total quantity') unless used <= quantity
+    errors.add(:used, 'can not be higher than total quantity') if used.present? && quantity.present? && used > quantity
```

Comment:
:+1:  Yes OK let's leave it like that, otherwise we would need to alter the validation statement